### PR TITLE
Update channels.csv

### PR DESCRIPTION
**Pull request by Dexer125**

### CODE REVIEW

File: data/channels.csv [hunk 1]
```diff
@@ -6221,6 +6222,7 @@ FilmBoxPremiumRomania.us,FilmBox Premium Romania,,,US,,,c/US,eng,,FALSE,http://w
 FilmBoxRussia.us,FilmBox Russia,,,US,,,c/US,eng,,FALSE,https://spiintl.com/channels/filmbox-russia,https://www.magticom.ge/images/channels/MjAxOC8wOS8xMC9hYjVhYzYxZS1hOGQ4LTRmNWUtOWQ5My1mYzgyMmVlMTBiY2IuanBn.jpg
 FilmBoxStarsAdria.us,FilmBox Stars Adria,,,US,,,c/US,eng,,FALSE,https://spiintl.com/channels/filmbox-stars,https://mtel.ba/oec/images/tv_channels/b4d254c5fbbaa76bb05643dfcd7a149d.png
 FilmBoxStarsHungary.us,FilmBox Stars Hungary,,,US,,,c/US,eng,,FALSE,https://spiintl.com/channels/filmbox-stars,https://www.sms.cz/kategorie/televize/bmp/loga/velka/filmboxstars.png
+FilmBoxStarsCzechia.us,FilmBox Stars Czechia,,US,,,c/US,eng,,FALSE,https://spiintl.com/channels/filmbox-stars,https://www.sms.cz/kategorie/televize/bmp/loga/velka/filmboxstars.png
```
Comment (LaneSh4d0w) [inline data/channels.csv:6225]:
```suggestion
FilmBoxStarsCzechia.us,FilmBox Stars Czechia,,,US,,,c/US,eng,,FALSE,https://spiintl.com/channels/filmbox-stars,https://www.sms.cz/kategorie/televize/bmp/loga/velka/filmboxstars.png 

```

File: data/channels.csv [hunk 2]
```diff
@@ -3758,6 +3758,7 @@ ChristTV.zw,Christ TV,,,ZW,,,c/ZW,eng,,FALSE,http://www.christtv.co/,https://www
 CHRODT43.ca,CHRO-DT-43,,CTV 2,CA,CA-ON,Ottawa,s/CA-ON,eng,,FALSE,,https://cdn.tvpassport.com/image/station/100x100/ctv2-2018.png
 CHTV.cn,CHTV,,,CN,,,c/CN,zho,,FALSE,,https://www.lyngsat.com/logo/tv/cc/ch_tv_cn.png
 CHTV.hn,CHTV,,,HN,,,c/HN,spa,,FALSE,http://chtv.hn/,https://www.lyngsat.com/logo/tv/cc/ch_tv_hn.png
+ChuckTV.sk,Chuck TV,,,SK,,,c/SK,slk,,FALSE,https://www.sms.cz/kategorie/televize/bmp/loga/velka/chucktv.png
```
Comment (LaneSh4d0w) [inline data/channels.csv:3761]:
```suggestion
ChuckTV.sk,Chuck TV,,,SK,,,c/SK,slk,,FALSE,https://www.sms.cz/kategorie/televize/bmp/loga/velka/chucktv.png
```

File: data/channels.csv [hunk 3]
```diff
@@ -17751,7 +17753,7 @@ PremiereFCAlternativo1.br,Premiere FC Alternativo 1,,,BR,,,c/BR,por,,FALSE,,
 PremiereFCAlternativo2.br,Premiere FC Alternativo 2,,,BR,,,c/BR,por,,FALSE,,
 PremierFootball.ph,Premier Football,,,PH,,,c/PH,eng,,FALSE,http://tapdmv.com/PremierFootball.html,https://www.lyngsat.com/logo/tv/pp/premier-football-ph.png
 PremierLeagueTV.uk,Premier League TV,,,UK,,,c/UK,eng,,FALSE,,https://www.lyngsat.com/logo/tv/pp/premier_league_tv_uk.png
-PremierSport.sk,Premier Sport,,,SK,,,c/SK,slk,,FALSE,http://www.premiersport.sk/,https://www.sms.cz/kategorie/televize/bmp/loga/velka/
+PremierSport1.sk,Premier Sport 1,,,SK,,,c/SK,slk,,FALSE,http://www.premiersport.sk/,https://www.sms.cz/kategorie/televize/bmp/loga/velka/
```
Comment (LaneSh4d0w) [inline data/channels.csv:17756]:
```suggestion
PremierSport1.sk,Premier Sport 1,,,SK,,,c/SK,slk,,FALSE,http://www.premiersport.sk/,https://www.sms.cz/kategorie/televize/bmp/loga/velka/
```

File: data/channels.csv [hunk 4]
```diff
@@ -20077,6 +20079,7 @@ SkyWorld3.th,SkyWorld 3,,,TH,,,c/TH,tha,,FALSE,,
 SL1.ir,SL 1,,,IR,,,c/IR,fas,movies,FALSE,,https://i.imgur.com/n9cuM2W.jpg
 SL2.ir,SL 2,,,IR,,,c/IR,fas,movies,FALSE,,https://i.imgur.com/BVuUx5R.jpg
 SlagerTV.hu,Sláger TV,,,HU,,,c/HU,hun,,FALSE,https://slagertv.tv/,https://www.sms.cz/kategorie/televize/bmp/loga/velka/slager.png
+SlagrOriginal.cz,Slágr Original,,CZ,,,c/CZ;c/SK,ces,,FALSE,,https://www.sms.cz/kategorie/televize/bmp/loga/velka/slagroriginal.png
```
Comment (LaneSh4d0w) [inline data/channels.csv:20082]:
```suggestion
SlagrOriginal.cz,Slágr Original,,,CZ,,,c/CZ;c/SK,ces,,FALSE,,https://www.sms.cz/kategorie/televize/bmp/loga/velka/slagroriginal.png
```

**Comment by Dexer125** [comment]
Now it's okay. So sorry for this mess. Changing .csv is pain. I will make it better next time.

**Pull request merged**